pkgs/by-name/ry/ryzen_monitor_ng/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "ryzen_monitor_ng";
```

Comment:
`ryzen-monitor-ng` is preferred over `ryzen_monitor_ng`, see [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming).

Reply:
My understanding of `pname` is that it should match upstream:

https://github.com/NixOS/nixpkgs/blob/57e6b3a9e4ebec5aa121188301f04a6b8c354c9b/pkgs/README.md?plain=1#L349

For all file names and paths, dashes should be used:

https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#code-conventions


Upstream uses `ryzen_monitor_ng`, so I have set `pname=ryzen_monitor_ng` and used `ryzen-monitor-ng` for all file and path names, and for the entries in `all-packages.nix`.

Reply:
However, I see I have used underscores in the nixos modules and `linux_kernels.nix`. Do you recommend changing those?

Reply:
> However, I see I have used underscores in the nixos modules and `linux_kernels.nix`. Do you recommend changing those?

Changing existing names is always harder than new ones.

> My understanding of `pname` is that it should match upstream:

That mainly focuses on the name. We should not name this package ``amd-monitoring-new``

Reply:
Thanks for clarifying, I'll change underscores to hyphens.